packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -64,6 +64,29 @@ const drawText = (txt) => {
   return svgLabel;
 };
 
+/**
+ * Searches for the closest parent from the parents list passed as argument.
+ * The parents list comes from an individual commit. The closest parent is actually
+ * the one farther down the graph, since that means it is closer to its child.
+ *
+ * @param {string[]} parents
+ * @returns {string}
+ */
+const findClosestParent = (parents) => {
+  let closestParent = '';
```

Comment:
Maybe it is better to return undefined when parent is missing

Reply:
The way I used `findClosestParent` in the code guarantees the function returns a parent as I mentioned above. But I do agree to return `undefined` for correctness, and the function could be used in other contexts aswell.